third_party/FMUParser/fmumini.c

```diff
@@ -520,7 +520,10 @@ int unpackmz(const char *filNam, char *tmpPat)
 
     printfDebug("%s opened\n", filename_try);
 
-    cmd = calloc(strlen(dirname) + 11, sizeof(char));
+    // the new string will be `mkdir -p "%s"`, so num characters added:
+    //                         1234567890  1
+    // appears to be 11.  However, sprintf puts a null terminator, so you need one more, making it 12
+    cmd = calloc(strlen(dirname) + 12, sizeof(char));
```

Comment:
If you look below, you'll see on Linux it is this:

`sprintf(cmd, "mkdir -p \"%s\"", dirname);`

I think I made my case pretty clear, and bumping it up one size allows the string to fit and run without the overflow.  

But also looking below, I see the Windows command is slightly shorter, without the `-p `, so it should probably be 9!?  This is so very fun!